pgpainless-core/src/main/java/org/pgpainless/util/selection/userid/SelectUserId.java

```diff
@@ -181,7 +181,7 @@ protected boolean accept(String userId) {
     }
 
     /**
-     * Filter that filters for user-ids which pass at least one of the given <pre>filters</pre>>.
```

Comment:
Good catch. Can you put this into its own commit?

Reply:
Done.